code/modules/jobs/job_types/cargo_service.dm

```diff
@@ -267,8 +267,8 @@ Janitor
 
 	outfit = /datum/outfit/job/janitor
 
-	access = list(access_janitor, access_maint_tunnels)
-	minimal_access = list(access_janitor, access_maint_tunnels)
+	access = list(access_janitor)
+	minimal_access = list(access_janitor)
```

Comment:
Janitors have maintenance access so they can clean up the oil, broken glass, and dried blood; as well as setting mouse traps.